packages/relay/tests/lib/services/hbarLimitService/hbarLimitService.spec.ts

```diff
@@ -494,6 +494,29 @@ describe('HBAR Rate Limit Service', function () {
         expect(result).to.be.false;
       });
     });
+
+    describe('It should be able to disable the rate limiter', function () {
+      const hbarLimitServiceDisabled = new HbarLimitService(
+        hbarSpendingPlanRepositoryStub,
+        ethAddressHbarSpendingPlanRepositoryStub,
+        ipAddressHbarSpendingPlanRepositoryStub,
+        logger,
+        register,
+        Hbar.fromTinybars(0),
+        limitDuration,
+      );
+      it('should return false if the rate limiter is disabled by setting HBAR_RATE_LIMIT_TINYBAR to zero', async function () {
+        // hbarLimitServiceDisabled.disableRateLimiter();
+        const result = await hbarLimitServiceDisabled.shouldLimit(
```

Comment:
This actually doesn't really test that the rate limiter is working.
For instance if a future code change was made and the return was false for some other reason that a 0 value this logic would not catch it.
I think we should reconsider the layout on the product side.

Reply:
Updated test to include a condition that would normally trigger rate limiting, but verifies that `shouldLimit` returns false since rate limiting is not enabled.